index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<main id="main">
+    <h1 id="title">Dr Norman borlaugh</h1>
+    <p>The man who saved a billion lives
+
+    </p>
```

Comment:
I suggest replacing p tag with h2 tag because HTML headings help search engines and users quickly scan and understand the page.